docs/cheatsheet/syntax.rst

```diff
@@ -0,0 +1,40 @@
+.. _ref_cheatsheet_syntax:
+
+Special Syntax
+==============
+
+There are several kinds of strings:
+
+.. code-block:: edgeql-repl
+
+    db> SELECT 'A simple string';
+    {'A simple string'}
+    db> SELECT "Also a simple string";
+    {'Also a simple string'}
+    db> SELECT '\'escaped quotes\'';
+    {"'escaped quotes'"}
+    db> SELECT r"Raw string where escapes aren\'t special";
+    {"Raw string where escapes aren\'t special"}
+    db> SELECT 'A
+    ...     multi line
+    ...     string';
+    {'A
+        multi line
+        string'}
+    db> SELECT 'A \
+    ...     string with \
+    ...     line continuation';
+    {'A string with line continuation'}
+
+It's possible to quote odd identifiers:
```

Comment:
Mention link properties here, `@` is odd.
Also mention decimal/bigint literals.

Reply:
Again, I'd need a link to the actual type definition here, i.e. a link to the types cheatsheet. Right?

Reply:
Sure